page 20980
/bbs/Gossiping/M.1495597324.A.D32.html download to ptt/Gossiping/20980_0.txt, time: 6.450000
/bbs/Gossiping/M.1495597338.A.FAF.html download to ptt/Gossiping/20980_1.txt, time: 4.070000
/bbs/Gossiping/M.1495597338.A.ED1.html download to ptt/Gossiping/20980_2.txt, time: 9.030000
/bbs/Gossiping/M.1495597341.A.AAF.html download to ptt/Gossiping/20980_3.txt, time: 7.130000
/bbs/Gossiping/M.1495597343.A.761.html download to ptt/Gossiping/20980_4.txt, time: 5.710000
/bbs/Gossiping/M.1495597369.A.660.html download to ptt/Gossiping/20980_5.txt, time: 4.890000
/bbs/Gossiping/M.1495597369.A.424.html download to ptt/Gossiping/20980_6.txt, time: 4.890000
/bbs/Gossiping/M.1495597388.A.046.html download to ptt/Gossiping/20980_7.txt, time: 4.180000
/bbs/Gossiping/M.1495597424.A.656.html download to ptt/Gossiping/20980_8.txt, time: 7.150000
/bbs/Gossiping/M.1495597503.A.B3B.html download to ptt/Gossiping/20980_9.txt, time: 6.290000
/bbs/Gossiping/M.1495597537.A.E66.html download to ptt/Gossiping/20980_10.txt, time: 4.560000
/bbs/Gossiping/M.1495597572.A.5CB.html download to ptt/Gossiping/20980_11.txt, time: 4.660000
/bbs/Gossiping/M.1495597574.A.648.html download to ptt/Gossiping/20980_12.txt, time: 3.600000
/bbs/Gossiping/M.1495597589.A.B5F.html download to ptt/Gossiping/20980_13.txt, time: 4.380000
/bbs/Gossiping/M.1495597589.A.607.html download to ptt/Gossiping/20980_14.txt, time: 3.410000
/bbs/Gossiping/M.1495597589.A.D22.html download to ptt/Gossiping/20980_15.txt, time: 5.310000
/bbs/Gossiping/M.1495597640.A.0FA.html download to ptt/Gossiping/20980_16.txt, time: 5.400000
/bbs/Gossiping/M.1495597676.A.020.html download to ptt/Gossiping/20980_17.txt, time: 4.780000
/bbs/Gossiping/M.1495597719.A.CC3.html download to ptt/Gossiping/20980_18.txt, time: 4.380000
/bbs/Gossiping/M.1495597751.A.1EB.html download to ptt/Gossiping/20980_19.txt, time: 2.990000
page 20979
/bbs/Gossiping/M.1495596841.A.971.html download to ptt/Gossiping/20979_0.txt, time: 3.490000
/bbs/Gossiping/M.1495596867.A.E27.html download to ptt/Gossiping/20979_1.txt, time: 3.170000
/bbs/Gossiping/M.1495596889.A.3E4.html download to ptt/Gossiping/20979_2.txt, time: 3.750000
/bbs/Gossiping/M.1495596891.A.262.html download to ptt/Gossiping/20979_3.txt, time: 3.760000
/bbs/Gossiping/M.1495596895.A.57D.html download to ptt/Gossiping/20979_4.txt, time: 5.410000
/bbs/Gossiping/M.1495596928.A.1C9.html download to ptt/Gossiping/20979_5.txt, time: 4.740000
/bbs/Gossiping/M.1495596949.A.043.html download to ptt/Gossiping/20979_6.txt, time: 6.000000
/bbs/Gossiping/M.1495596965.A.836.html download to ptt/Gossiping/20979_7.txt, time: 3.480000
/bbs/Gossiping/M.1495596969.A.019.html download to ptt/Gossiping/20979_8.txt, time: 4.390000
/bbs/Gossiping/M.1495597022.A.500.html download to ptt/Gossiping/20979_9.txt, time: 3.770000
/bbs/Gossiping/M.1495597050.A.9D5.html download to ptt/Gossiping/20979_10.txt, time: 6.470000
/bbs/Gossiping/M.1495597055.A.63C.html download to ptt/Gossiping/20979_11.txt, time: 4.140000
/bbs/Gossiping/M.1495597077.A.CEF.html download to ptt/Gossiping/20979_12.txt, time: 6.650000
/bbs/Gossiping/M.1495597107.A.EF2.html download to ptt/Gossiping/20979_13.txt, time: 5.030000
/bbs/Gossiping/M.1495597130.A.824.html download to ptt/Gossiping/20979_14.txt, time: 5.200000
/bbs/Gossiping/M.1495597162.A.A05.html download to ptt/Gossiping/20979_15.txt, time: 4.770000
/bbs/Gossiping/M.1495597165.A.9A7.html download to ptt/Gossiping/20979_16.txt, time: 5.200000
/bbs/Gossiping/M.1495597210.A.8E0.html download to ptt/Gossiping/20979_17.txt, time: 4.190000
/bbs/Gossiping/M.1495597240.A.F37.htmlTraceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 438, in send
    timeout=timeout
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 649, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/util/retry.py", line 357, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 386, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/packages/urllib3/connectionpool.py", line 382, in _make_request
    httplib_response = conn.getresponse()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 1331, in getresponse
    response.begin()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 297, in begin
    version, status, reason = self._read_status()
  File "/home/studentyl/anaconda3/lib/python3.6/http/client.py", line 266, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
requests.packages.urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./crawler.py", line 177, in <module>
    main(sys.argv[1:])
  File "./crawler.py", line 158, in main
    downloadForum(forumName, startPage, pages)
  File "./crawler.py", line 140, in downloadForum
    downloadPage(str(articleCount), forumName, pageNum, 'https://www.ptt.cc'+link)
  File "./crawler.py", line 82, in downloadPage
    pageText = getPageText(link)
  File "./crawler.py", line 13, in getPageText
    pageText=requests.get(link, cookies={'over18':'1'}).text
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 72, in get
    return request('get', url, params=params, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 518, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/sessions.py", line 639, in send
    r = adapter.send(request, **kwargs)
  File "/home/studentyl/anaconda3/lib/python3.6/site-packages/requests/adapters.py", line 488, in send
    raise ConnectionError(err, request=request)
requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response',))
